docs/resources/cluster_aks.md

```diff
@@ -12,7 +12,7 @@ description: |-
 ## Example Usage
 
 ```terraform
-data "spectrocloud_cloudaccount_aws" "account" {
+data "spectrocloud_cloudaccount_azure" "account" {
```

Comment:
@paraglade good catch, I had to change template though, please check my latest commit